web/src/components/apps/AppDetailPage.tsx

```diff
@@ -392,6 +374,37 @@ function AppDetailPage(props: Props) {
     });
   };
 
+  // add types
```

Comment:
```suggestion
  // add types
```
is this still needed?